ci-scripts/hotfix-publish.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -u -e
+
+git config --global user.email "fundamental@sap.com"
```

Comment:
@JKMarkowski How about using an environment variable here?

Reply:
@Betrozov sure, done  could you validate it? I'm not very into sh scripts

Reply:
@JKMarkowski I'm talking about to use the Travis environment variable :slightly_smiling_face: 
and for it we not need to create `.ci-env/github.sh` we can just to user environment variable that will be located on Travis

Reply:
Ok, It makes more sense :laughing: 
![image](https://user-images.githubusercontent.com/26483208/100454168-a7ac3d80-30bc-11eb-8115-4f0541a9a761.png)

Reply:
Great job 👍
Thanks 🙂